README.md

```diff
@@ -1,14 +1,20 @@
-[![Version](https://img.shields.io/badge/version-8.2.0-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
+[![Version](https://img.shields.io/badge/version-9.0.0-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
 [![Docs](https://img.shields.io/badge/docs-website-yellow.svg)](https://0x-org.gitbook.io/mesh)
 [![Chat with us on Discord](https://img.shields.io/badge/chat-Discord-blueViolet.svg)](https://discord.gg/HF7fHwk)
 [![Circle CI](https://img.shields.io/circleci/project/0xProject/0x-mesh/master.svg)](https://circleci.com/gh/0xProject/0x-mesh/tree/master)
 
 # 0x Mesh
 
 0x Mesh is a peer-to-peer network for sharing orders that adhere to the
-[0x order message format](https://0x.org/docs/guides/v2-specification#order-message-format).
+[0x order message format](https://0x.org/docs/guides/v3-specification#orders).
+
+## Project status
+
+We have reached the point where Mesh is being used by some teams in production. We feel that for many use cases, Mesh is stable enough for production. However, we caution that there are some issues and shortcomings in its current state, which generally fall into two categories:
+
+- Order sharing: Mesh uses a significant amount of bandwidth for it's current order sharing algorithm and we intend on improving its efficiency by an order of magnitude. Until then, we advise you host your node on [Digital Ocean](https://www.digitalocean.com/) where it's hosting costs are cheaper (~$25 mo). We also know that orders are not propagating through the network as quickly as we would like. We have a lot of ideas for how to optimize order propagation speed in the future and there should be major improvements coming soon. See [#594](https://github.com/0xProject/0x-mesh/issues/594), [#551](https://github.com/0xProject/0x-mesh/issues/551), and [#638](https://github.com/0xProject/0x-mesh/issues/638).
```

Comment:
Hopefully we can remove this in the next release!

Reply:
Hmm.. we could almost remove it now. But maybe it's better to measure and confirm.

Reply:
I think we should measure and confirm too